script.js

```diff
@@ -1,19 +1,19 @@
-const display = document.querySelector("#display-data-done-main");
+const display = document.querySelector(".display-data-done-main");
```

Comment:
Related to the change in the HTML file, when querying for a element by its CSS class, you want to use a dot in front of the class name. If you were querying by an ID, then you would use the hashtag.